web/settings.py

```diff
@@ -160,8 +160,13 @@ def configure():
 
 
 Settings.configure()
+log_level_str = Settings.app['log_level']
+env_log_level_str = os.environ.get("SANICAPP_WORKERS_LOG_LEVEL", "None")
```

Comment:
technically, we could leave default value empty, which would supply `None` instead of passing `"None"`.